spec/support/v1_helper.rb

```diff
@@ -0,0 +1,5 @@
+module V1Helper
```

Comment:
why the file name is asymmetric? should it be `v1x0_helper.rb`?

Reply:
https://github.com/RedHatInsights/approval-api/pull/278#issuecomment-593497675